shared/src/mobileMain/kotlin/dev/johnoreilly/confetti/ConferencesViewModel.kt

```diff
@@ -62,7 +64,8 @@ open class ConferencesViewModel : KMMViewModel(), KoinComponent {
         }
     }
 
-    suspend fun setConference(conference: String) {
+    open suspend fun setConference(conference: String) {
```

Comment:
Curious why this needs to be open. Does Koin or something else extend the ViewModel ?

Reply:
Not sure specifically about this case but KMMViewModel (https://github.com/rickclephas/KMM-ViewModel) does support view model being subcassed for example in Swift code but not sure if it's a general requirement